.travis.d/deploy_docs.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+cd $TRAVIS_BUILD_DIR
+git fetch origin gh-pages:origin/gh-pages
+git checkout origin/gh-pages -b gh-pages || git checkout --orphan gh-pages
+git reset
+ls | grep -v 'docs' | xargs rm -rf
+mv docs/html/* .
+rm -rf docs
+git add *
```

Comment:
i want to exclude hidden files(startswith . files)
`git add -A` will add all files include hidden file.